kyuubi-server/pom.xml

```diff
@@ -272,6 +272,11 @@
             <artifactId>sqlite-jdbc</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
```

Comment:
the LICENSE-binary and NOTICE-binary need to be updated since this new jar will be shipped by the binary tarball